athena-vertica/src/main/java/com/amazonaws/athena/connectors/vertica/VerticaMetadataHandler.java

```diff
@@ -174,6 +203,79 @@ public ListTablesResponse doListTables(BlockAllocator allocator, ListTablesReque
         return new ListTablesResponse(request.getCatalogName(), tables, null);
 
     }
+    protected ArrowType getArrayArrowTypeFromTypeName(String typeName, int precision, int scale)
+    {
+        // Default ARRAY type is VARCHAR.
+        return new ArrowType.Utf8();
+    }
+    @Override
+    public GetDataSourceCapabilitiesResponse doGetDataSourceCapabilities(BlockAllocator allocator, GetDataSourceCapabilitiesRequest request)
+    {
+        ImmutableMap.Builder<String, List<OptimizationSubType>> capabilities = ImmutableMap.builder();
+        queryPassthrough.addQueryPassthroughCapabilityIfEnabled(capabilities, configOptions);
+
+        return new GetDataSourceCapabilitiesResponse(request.getCatalogName(), capabilities.build());
+    }
+
+
+    @Override
+    public GetTableResponse doGetQueryPassthroughSchema(final BlockAllocator blockAllocator, final GetTableRequest getTableRequest)
```

Comment:
The implementation of this method brings up few issues;

1. We are using JDBC Implementation here; (almost copy/paste from the JDBC Metadatahandler) whereas `doGetTable` uses `VerticaSchemaUtils` class. 
2. This brings up an important question; why are we not using the JDBC metadatahandler to begin with? If there is a good reason for it, then we shouldn't be using the JDBC implementation here either.
3. If it seems that there in fact we don't need to have different implementation between JDBC metadata handler and Vertica's, we need to use this opportunity to unify the code.

Reply:
incorporated the changes for vertica datatype to arrow in dogetQuerypassthroughSchema.